firmware/Core/Inc/Common.h

```diff
@@ -15,6 +15,8 @@
 
 #define CTRL_LOOP_FREQUENCY 50  // Hz
 
+#define LOG_SUCCEEDED                 1
+#define LOG_FAILED                    0
```

Comment:
Feel free to just use the global RETROFRIDGERATION_SUCCEEDED /FAILED macros. No need for each module to have it's own macros defined.